lib/debuglet.js

```diff
@@ -434,3 +441,53 @@ Debuglet.mapSubtract = function mapSubtract(A, B) {
   return removed;
 };
 
+/**
+ * Formats the message base with placeholders `$0`, `$1`, etc
+ * by substituting the provided expressions. If more expressions
+ * are given than placeholders extra expressions are dropped.
+ */
+Debuglet.format = function(base, exprs) {
```

Comment:
Although, if this works, I am not too motivated to make you change it. I just need to convince myself that it works.